src/Sarif.Converters/AndroidStudioConverter.cs

```diff
@@ -123,8 +123,12 @@ public Result ConvertProblemToSarifResult(AndroidStudioProblem problem)
 
             SetSarifResultPropertiesForProblem(result, problem);
             var location = new Location();
-            location.FullyQualifiedLogicalName = AddLogicalLocationEntriesForProblem(problem, out int logicalLocationIndex);
-            location.LogicalLocationIndex = logicalLocationIndex;
+            location.LogicalLocation = new LogicalLocation
```

Comment:
>new LogicalLocation [](start = 39, length = 19)

anywhere u see this pattern where i create a nw objecxt without checkin gif its needed, i will fix. this introduces empty node in the json.